pyproject.toml

```diff
@@ -76,6 +75,7 @@ nh3 = "^0.2.14"
 retry2 = "^0.9.5"
 pluggy = "^1.3.0"
 named-enum = "^1.4.0"
+drf-nested-routers = "^0.93.5"
```

Comment:
Just curious, what motivated switching to this rather than `drf-extensions`?

Reply:
I switched out the library because drf-extentions didn't support having parameter names that are different from the model parameters without a bunch of custom code

Which makes sense, because unless you are using drf-spectacular to auto generate documentation, the parameter names in the urls are not visible to the user in any way